tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -147,7 +148,13 @@ bool VersionSupportInfo::SkipDxil_1_1_Test() {
   }
   return false;
 }
-
+bool VersionSupportInfo::SkipOutOfMemoryTest() {
+  if (m_CompilerPreservesBBNames) { // same detection logic at the moment
```

Comment:
Should probably just rename this member to m_CompilerIsDebugBuild or something, since that's what it really tracks.